### PR TITLE
Approving Speed Stacks Stackmat timer Generation 5

### PR DESCRIPTION
**Pull request by Ivan-Brigidano**
See the [WCA Forum discussion](https://www.google.com/url?q=https://forum.worldcubeassociation.org/t/wssa-g5-timer-discussion/12695&sa=D&source=docs&ust=1635080708434000&usg=AOvVaw3b2DzNiCWQqdn1ZusQ9ANB).

### CODE REVIEW

**Comment by xsrvmy** [comment]
Should there be an explicit regulation that says the competitor must be allowed to use 2 pad mode (ie. organizer can't require 4 pad mode)?

File: wca-guidelines.md [hunk 1]
```diff
@@ -106,6 +106,7 @@ To be more informative, each Guideline is classified using one of the following
 ## <article-7><environment><environment> Article 7: Environment
 
 - 7f1a+) [CLARIFICATION] A full-size mat should have minimum dimensions of 30cm (left to right) by 25cm (front to back).
+- 7f1a++) [ADDITION] Generation 5 Pro Timer should be used in 2-pads mode.
```
Comment (Rouxles) [inline wca-guidelines.md]: I think 2-pad is the correct term that Speedstacks uses
```suggestion
- 7f1a++) [ADDITION] Generation 5 Pro Timer should be used in 2-pad mode.
```

Comment (Jambrose777) [inline wca-guidelines.md]:
```suggestion
- 7f1a++) [ADDITION] Generation 5 Pro Timer should be used in 2-pad mode.
```
I believe this is referred to as 2-pad mode without the s everywhere on speedstacks

**Comment by hanwu85** [comment]
If the timer is set in 4-pad mode by accident, I think there should be an additional regulation explaining the outcome.
1.  The competitor must notify the judge before starting the solve in order to get an extra attempt.
2. The competitor can accept the result if he want to.

Before the attempt starts.
The judge should check the timer before inspection phase.

I will come up with better sentence to put into the regulations.

**Review by mirandapedro — APPROVED** [review]
LGTM!

File: wca-regulations.md [hunk 2]
```diff
@@ -198,12 +198,13 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 - 7e) The competition area must be smoke-free.
 - 7f) Solving station:
     - 7f1) Definitions:
-        - 7f1a) Stackmat: The Speed Stacks Stackmat timer and a full-size compatible mat. Generation 2, Generation 3 Pro Timer, and/or Generation 4 Pro Timer must be used for time measurement.
+        - 7f1a) Stackmat: The Speed Stacks Stackmat timer and a full-size compatible mat. Generation 2, Generation 3 Pro Timer, Generation 4 Pro Timer and/or Generation 5 Pro Timer must be used for time measurement.
```
Comment (Rouxles) [inline wca-regulations.md]:
```suggestion
        - 7f1a) Stackmat: The Speed Stacks Stackmat timer and a full-size compatible mat. Generation 2, Generation 3 Pro Timer, Generation 4 Pro Timer, and/or Generation 5 Pro Timer must be used for time measurement.
```

We should keep the Oxford comma there to keep it consistent with the previous version

**Review by Rouxles — APPROVED** [review]

File: wca-regulations.md [hunk 3]
```diff
@@ -379,7 +380,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 - A3) Inspection:
     - A3a) The competitor may inspect the puzzle at the start of each attempt.
         - A3a1) The competitor is allotted a maximum of 15 seconds to inspect the puzzle and start the solve.
-    - A3b) The judge prepares the timer by turning it on and resetting if necessary. Separately, the judge also prepares a stopwatch for timing inspection (partially superseded by [Regulation Y7d1](regulations:regulation:Y7d1)).
+    - A3b) The judge prepares the timer by turning it on and resetting if necessary. If a Generation 5 Pro Timer is in use, the judge must ensure it is on 2-pad mode. Separately, the judge also prepares a stopwatch for timing inspection (partially superseded by [Regulation Y7d1](regulations:regulation:Y7d1)).
```
Comment (lgarron) [inline wca-regulations.md]: Small wording suggestion.
```suggestion
    - A3b) The judge prepares the timer by turning it on and resetting if necessary. If a Generation 5 Pro Timer is in use, the judge must ensure it is set to 2-Pad mode. Separately, the judge also prepares a stopwatch for timing inspection (partially superseded by [Regulation Y7d1](regulations:regulation:Y7d1)).
```

Speed Stacks themselves seems to call it "2-Pad mode" with a capital P.

**Review by lgarron — APPROVED** [review]
As discussed within the WRC in private, I am not in favor of allowing Gen 5 timers at this point. However, given that the team as a whole is in favor, this change looks good!

**Pull request merged**